<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotPixels</class>
 <widget class="QDialog" name="PlotPixels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Pixels - EMPAD Browse prebeta</string>
  </property>
  <widget class="QCustomPlot" name="pixelPlot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>781</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>60</y>
     <width>93</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_Analog">
      <property name="text">
       <string>Analog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Digital">
      <property name="text">
       <string>Digital</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Combined">
      <property name="text">
       <string>Combined</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_backSub">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Background Subtracted</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>751</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Pixel x:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_PixX"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pixY"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>start frame:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_StartFrame"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>end frame:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_EndFrame"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_collectVals">
      <property name="text">
       <string>Collect Values</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_showEven">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>560</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Even</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_showOdd">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>590</y>
     <width>78</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Odd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_expData">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>300</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Data</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
